packages/uniforms-material/__tests__/index.ts

```diff
@@ -47,6 +47,9 @@ describe('@RTL', () => {
     getListAddField: screen => screen.getByText(/\+/),
   });
   suites.testLongTextField(theme.LongTextField);
+  suites.testNestField(theme.NestField, {
+    skipForMui: true,
+  });
```

Comment:
1. If possible, keep them one line.
2. Can this option be more specific, i.e., say what is skipped exactly?